src/Slider.js

```diff
@@ -0,0 +1,235 @@
+import React, { PureComponent } from "react";
+import { View, Animated, StyleSheet, PanResponder } from "react-native";
+import PropTypes from "prop-types";
+import GalioTheme, { withGalio } from './theme';
+
+
+class Slider extends PureComponent {
```

Comment:
There is a ReactNative component named [Slider](https://facebook.github.io/react-native/docs/slider) ... which is bound to the native component from iOS and Android.

Reply:
Could you please specify more exactly what are you referring to? I know about shouldComponentUpdate() as I've read about PureComponent before choosing it. I know about the Slider as well but I've created a new one because the native Slider isn't as customizable as it needs to be for Galio's purpose.

Reply:
What does the method shouldComponentUpdate() do?